static/src/javascripts/projects/common/modules/social/share-count.js

```diff
@@ -46,15 +46,16 @@ define([
     }
 
     function addToShareCount(val) {
-        if ($shareCountEls.hasClass('js-sharecount-immersive')) {
-            shareCountTemplate = template(shareCountImmersiveTemplate, {
-                icon: svgs('share')
-            });
-        }
+        var shareSvg = svgs('share');
+        var shareTemplate = $shareCountEls.hasClass('js-sharecount-immersive') ? shareCountImmersiveTemplate : shareCountTemplate;
```

Comment:
Eventually this condition could be update to `window.guardian.config.page.isImmersive`